fusedrug/data/protein/structure/split_scfv_chain.py

```diff
@@ -0,0 +1,180 @@
+from jsonargparse import CLI
+from fusedrug.data.protein.structure.structure_io import (
+    load_pdb_chain_features,
+    save_structure_file,
+)
+from typing import Optional, Sequence
+from os.path import isfile, join, dirname, basename
+import os
+import sys
+import subprocess
+import threading
+
+
+def main(
+    *,
+    input_pdb_path: str,
+    input_scfv_chain_id: str,
+    output_pdb_path_extensionless: str,
+    output_heavy_chain_id: Optional[str] = "H",
+    output_light_chain_id: Optional[str] = "L",
+    passthrough_chains: Optional[str] = None,
+    cleanup_temp_files: bool = True,
+) -> None:
+    """
+
+    Takes an input PDB files and splits it into separate files, one per describe chain, allowing to rename the chains if desired
```

Comment:
Is it a copied description?  
Is it only relevant to complexes with antibodies?
Does it assume that input heavy chain id is H and light is L?
What does it mean output_pdb_path_extensionless? and passthrough_chains?